platform/source/mbed_retarget.cpp

```diff
@@ -554,12 +651,18 @@ extern "C" int open(const char *name, int oflag, ...)
 
     return fildes;
 }
+#endif // !(MBED_CONF_PLATFORM_STDIO_MINIMAL_CONSOLE_ONLY)
 
 extern "C" int PREFIX(_close)(FILEHANDLE fh)
 {
+#if !(MBED_CONF_PLATFORM_STDIO_MINIMAL_CONSOLE_ONLY)
     return close(fh);
+#else
+    return 0;
+#endif // !(MBED_CONF_PLATFORM_STDIO_MINIMAL_CONSOLE_ONLY)
 }
 
+#if !(MBED_CONF_PLATFORM_STDIO_MINIMAL_CONSOLE_ONLY)
```

Comment:
Remove also the declaration in mbed_retarget.h

Reply:
[Done](https://github.com/hugueskamba/mbed-os/compare/9ab95d568e3b1b18b96f057a9ddf6a43bbeaada9..984d3d625c22b050e42c283372303827d347f0d4).